css/main.css

```diff
@@ -86,31 +107,35 @@ h2 {
   font-size: 18px;
   color: #fff;
 }
+.btn-sec-1:hover,
+.btn-sec-2:hover,
+.btn-sec-3:hover,
+.btn-sec-5:hover,
+.download-link:hover,
+.btn-sec-1:focus,
+.btn-sec-2:focus,
+.btn-sec-3:focus,
+.btn-sec-5:focus,
+.download-link:focus {
+  box-shadow: 2px 6px 15px rgba(11, 68, 253, 0.35);
+  transition: 0.2s;
+}
```

Comment:
Why do you need so many modifiers?  Can we use common class for all the elements.